### DISCUSSION TITLE
Mesa Dev Session 15 APR

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1621] [General]
When: 15 April --- 2000 UTC+8/1400 CET/1200 GMT/0800 EDT

Where: [via Matrix](https://matrix.to/#/#mesa-dev:matrix.org)

Proposed Agenda:

GSOC
Mesa 2.0
Mesa-Geo

@jackiekazil @rht @wang-boyu @Tortar @EwoutH Please feel free to add or takeaway from this agenda

**Comment by tpike3** [comment]
Updated the dev session date based on discussion

**Reply by EwoutH** [reply]
Noted! Will try to attend

**Comment by tpike3** [comment]
Dev Notes: 

- @jackiekazil will be at PyCON week of 24 April and will be using Sprints to help advance Mesa. If there are issues or pull request you want them focus on please add the *sprint* label -- see [matrix chat](https://matrix.to/#/#project-mesa:matrix.org)

- We endorsed one project for GSoC, now we are just waiting to see if it gets picked up at the beginning of May

- We talked about running a SIG at [CSSSA conference](https://computationalsocialscience.org/conferences/css2023/) and an ABM - GIS panel at next years [AAG conference](https://www.aag.org/)
